index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Diego Vidal</title>
+    <link rel="stylesheet" href="style.css">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <!-- <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap" rel="stylesheet"> -->
+
+</head>
+
+<body>
+    <div class="body-wrap">
+
+        <div class="nav-flex-container">
+            <div class="nav-logo">My Logo</div>
+            <button class="nav-menu-button"><img class=" nav-logo" src="./icons/Icon - Menunav-button.png"
+                    alt="navegation button"></button>
+            <p>Portfolio</p>
+            <p>About</p>
+            <p>Contact</p>
+        </div>
+
+        <header class="header-container">
+
+            <section class="header-text">
+                <h1>
+                    I’m Alexandra. <br>
+                    Glad to see you!
+                </h1>
+                <p>
+                    I’m a software developer! I can help you build a product ,
+                    feature or website Look through some of my work and
+                    experience! If you like what you see and have a project
+                    you need coded, don’t hesitate to contact me.
+                </p>
+                <p class="lets-conect">
+                    LET'S CONNECT
```

Comment:
To make layouting and styling easier, consider aligning the items within this section with flexbox.